Fonts/PTSenif-Bold/specimen_files/grid_12-825-55-15.css

```diff
@@ -0,0 +1,180 @@
+/* Notes about grid:
+Columns:      12
+Grid Width:   825px
+Column Width: 55px
+Gutter Width: 15px
+------------------------------- */
+
+.section { margin-bottom: 18px;
+}
+
+.section:after { content: ".";display: block;height: 0;clear: both;visibility: hidden;
+}
+
+.section { *zoom: 1;
+}
+
+.section .firstcolumn,
+.section .firstcol { margin-left: 0;
```

Comment:
Codacy found an issue: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)